docs/source/supported_layers.md

```diff
@@ -0,0 +1,183 @@
+| ZeroPadding 1D            | ×             | √            | √           | ×             | √                | √                    |
+| ZeroPadding 2D            | ×             | √            | √           | ×             | √                | √                    |
+| ZeroPadding 3D            | ×             | √            | √           | ×             | √                | √                    |
+| Im2Col                    | √             | ×            | ×           | ×             | ×                | ×                    |
+| Spatial Pyramid Pooling   | √             | ×            | ×           | √             | ×                | ×                    |
+* Upsampling in Caffe can be done by using methods shown [here](https://gist.github.com/tnarihi/54744612d35776f53278) 
+ ### Pooling Layers
+| Layer                     | Caffe         | Keras        | Tensorflow  | Fabrik Caffe  | Fabrik Keras    | Fabrik Tensorflow  |
+| :-----------------------: | :-----------: | :----------: | :---------: | :-----------: | :--------------: | :-----------------: | 
+| MaxPooling1D              | √             | √            | √           | √             | √               | ×                    |
+| MaxPooling2D              | √             | √            | √           | √             | √               | √                    |
+| MaxPooling3D              | √             | √            | √           | √             | √               | √                    |
+| AveragePooling1D          | √             | √            | √           | ×             | √               | ×                    |
+| AveragePooling2D          | √             | √            | √           | ×             | √               | ×                    |
+| AveragePooling3D          | √             | √            | √           | ×             | √               | ×                    |
+| GlobalMaxPooling1D        | ×             | √            | √           | ×             | ×               | ×                    |
+| GlobalAveragePooling1D    | ×             | √            | √           | ×             | ×               | ×                    |
+| GlobalMaxPooling2D        | ×             | √            | √           | ×             | ×               | ×                    |
+| GlobalAveragePooling2D    | ×             | √            | √           | ×             | ×               | ×                    |
+| GlobalMaxPooling3D        | ×             | √            | √           | ×             | ×               | ×                    |
+| GlobalAveragePooling3D    | ×             | √            | √           | ×             | ×               | ×                    |
+| Stochastic Pooling        | √             | ×            | ×           | √             | ×               | ×                    |
+ 
+ ### Data Layers
+| Layer                     | Caffe         | Keras        | Tensorflow  | Fabrik Caffe  | Fabrik Keras    | Fabrik Tensorflow  |
```

Comment:
Can you also tell me why we have 6 columns two for each layer?
I think only mentioning which layers for a framework does Fabrik support is good enough.